src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -320,5 +320,10 @@
   "stsjorbsmod:Feast": {
     "NAME": "Feast",
     "DESCRIPTION": "Remember stsjorbsmod:Gluttony. NL Draw !M! cards. NL Exhaust."
+  },
+  "stsjorbsmod:TimeWalk": {
+    "NAME": "Time Walk",
+    "DESCRIPTION": "Discard your hand and lose all Block. NL Gain !M! energy and draw !stsjorbsmod:UrMagic! cards. NL Gain !stsjorbsmod:MetaMagic! Weak. NL Exhaust.",
```

Comment:
Should "energy" be capitalized? I think the main classes have unique symbols for theirs, so they don't offer an exact comparison.

Reply:
We actually have the ability to use the energy symbol also! The placeholder you want for it is `[E]` (see `Channel`'s text for an example)